bon-macros/src/builder/builder_gen/member/config/mod.rs

```diff
@@ -212,15 +224,42 @@ impl MemberConfig {
             self.validate_mutually_allowed(
                 ParamName::StartFn,
                 self.start_fn.span(),
-                &[ParamName::Into],
+                &[ParamName::Into, ParamName::Getter],
             )?;
         }
 
         if self.finish_fn.is_present() {
             self.validate_mutually_allowed(
                 ParamName::FinishFn,
                 self.finish_fn.span(),
-                &[ParamName::Into],
+                &[ParamName::Into, ParamName::Getter],
+            )?;
+        }
+
+        if let Some(getter) = &self.getter {
+            if !cfg!(feature = "experimental-getter") {
+                bail!(
+                    &getter.key.span(),
+                    "`getter` attribute is experimental and requires \
+                    \"experimental-getter\" cargo feature to be enabled; \
+                    we would be glad to make this attribute stable if you find it useful; \
+                    please leave a 👍 reaction under the issue https://github.com/elastio/bon/issues/221 \
```

Comment:
```suggestion
                    please leave a 👍 reaction under the issue https://github.com/elastio/bon/issues/225 \
```

Reply:
Yeah, I update this code locally, although I can't push my changes to [Kindness-Works:laz/221-getters](https://github.com/Kindness-Works/bon/tree/laz/221-getters) even though there is this sentence in the PR metadata:
![image](https://github.com/user-attachments/assets/fc842b99-0657-4c02-b6ec-c310af52062e)

Maybe you have an org-wise rejection for writes from external contributors?

Anyway, I'll force-merge this PR and submit my amendments in a separate PR